src/config/unit_test/mod.rs

```diff
@@ -1,4 +1,13 @@
-#[cfg(all(test, feature = "vector-unit-test-tests"))]
+// should match vector-unit-test-tests feature
```

Comment:
For unit tests we just use the `default` feature. Theoretically we could also enable `vector-unit-test-tests` when running tests in CI, but what I have here seems to be a closer pattern to what we already do.